lib/nimbusRecipe.ts

```diff
@@ -92,6 +92,18 @@ export class NimbusRecipe implements NimbusRecipeType {
     branchInfo.template = template;
     branchInfo.surface = getSurfaceDataForTemplate(template).surface;
 
+    // Microsurvey check
+    if (
+      branchInfo.id.toLowerCase().includes("survey") ||
+      branchInfo.slug.toLowerCase().includes("survey") ||
+      (branchInfo.description &&
+        branchInfo.description.toLowerCase().includes("survey")) ||
+      (branchInfo.userFacingName &&
+        branchInfo.userFacingName.toLowerCase().includes("survey"))
+    ) {
+      branchInfo.isMicrosurvey = true;
+    }
+
```

Comment:
`getBranchInfo` is already too long, let's hoist this into a `_branchInfoHasMicrosurvey()` private method and call it from here.